pkg/features/kube_features.go

```diff
@@ -620,7 +620,7 @@ const (
 	SetHostnameAsFQDN featuregate.Feature = "SetHostnameAsFQDN"
 
 	// owner: @ksubrmnn
-	// alpha: v1.14
```

Comment:
Can you leave in //alpha: v1.14 like some of the more recent updates above to maintain some history here?